calblueprint/berkeley-student-cooperative
# Andrei/member view shift card

# ✨ New in this PR ✨

## One liner: what did you do? 🛠
basic shift card for member viewing.  Displays a shift's information, the assigned members, and whether they were verified. 
Verification to firebase pending.


## Coverage 🙆‍♀️
_use this section to break up your task into submodules and track progress_ 
- [x]  display the shift’s details at the top of the page
- [x]  show the list of members assigned to the shift and that member’s status on the shift
    - [x]  incomplete
    - [x]  complete —> shows timestamp under Time column
- [x]  verification PIN field (limit input to be only 5 digits long) – input must also be invisible (like dotted out and not showing the digits)
    - [ ]  would be cool if the digit shows for only a second or two when its first inputted, but then gets dotted out

## How can the reviewer test your code? 👩‍💻
Click on View Shift button.  Check that information matches a given shift (manually change the ID in the index.jsx file).  In firebase, update the verifiedShifts collection to see that status reflects if shift was verified.  To test if verifyShift works, check use a pin found in the house's pin map, check that non-existent pins or current user's pin can't verify.  In index, I left two sign in buttons (one is commented out),  can use those to test different users, feel free to add new ones though.


## Any bugs you encountered or still having trouble with? 🐛


## Resources 📔
_link online resources and related PRs_ 


🏚 cc: @gregoriiaaa
